shimmy/utils/meltingpot.py

```diff
@@ -18,26 +18,22 @@ def load_meltingpot(substrate_name: str):
         substrate_name: str
 
     Returns:
-        env: meltingpot.python.utils.substrates.substrate.Substrate
+        env: meltingpot.utils.substrates.substrate.Substrate
     """
-    import meltingpot.python
+    import meltingpot
     from ml_collections import config_dict
 
     # Create env config
     substrate_name = substrate_name
-    player_roles = meltingpot.python.substrate.get_config(
-        substrate_name
-    ).default_player_roles
+    player_roles = meltingpot.substrate.get_config(substrate_name).default_player_roles
```

Comment:
I think this is the problem, it should be utils. Make sure to run the pytests locally

Reply:
The tests pass locally, the version of mp used by the docker actions was an old version, so I updated it.